spec/spec_helper.cr

```diff
@@ -6,12 +6,32 @@ Granite::Connections << Granite::Adapter::Mysql.new(name: "mysql", url: ENV["MYS
 Granite::Connections << Granite::Adapter::Pg.new(name: "pg", url: ENV["PG_DATABASE_URL"])
 Granite::Connections << Granite::Adapter::Sqlite.new(name: "sqlite", url: ENV["SQLITE_DATABASE_URL"])
 
-Spec.before_each do
-  Granite.settings.default_timezone = Granite::TIME_ZONE
-end
-
 require "spec"
 require "../src/granite"
 require "../src/adapter/**"
 require "./spec_models"
 require "./mocks/**"
+
+Spec.before_suite do
+  Granite.settings.default_timezone = Granite::TIME_ZONE
+  {% if flag?(:spec_logs) %}
+    ::Log.builder.bind(
+      # source: "spec.client",
+      source: "*",
+      level: ::Log::Severity::Trace,
+      backend: ::Log::IOBackend.new(STDOUT, dispatcher: :sync),
+    )
+  {% end %}
+end
+
+Spec.before_each do
+  # I have no idea why this is needed, but it is.
+  Granite.settings.default_timezone = Granite::TIME_ZONE
```

Comment:
I wonder if this is something we should detect at run-time and use the timezone of the server by default instead of having to specify it.

I'm not sure, just an idea and noting it here in case anyone else has an opinion/thought

Reply:
agreed

Reply:
I'm going to say this is a sane default and let's move forward from here. However, it would be wise to surface some docs on this and maybe make configuring it easier (ie make it more obvious how to set it).